### PR TITLE
fix b2c challenge by passing in the policy

### PR DESCRIPTION
**Pull request by jennyf19**
also add in b2c blazorwasm for testing the challenge

### CODE REVIEW

File: tests/blazorwasm2-b2c-hosted/Client/Program.cs% [hunk 1]
```diff
@@ -0,0 +1,36 @@
+using System;
```
Comment (jmprieur) [inline tests/blazorwasm2-b2c-hosted/Client/Program.cs%:1]: We don't need this file (which was generated by the configuration tool)

**Review by jmprieur — COMMENTED** [review]
LGTM.
Thanks @jennyf19

File: tests/blazorwasm2-b2c-hosted/Server/Controllers/WeatherForecastController.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    {
+        private static readonly string[] Summaries = new[]
+        {
+            "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching"
+        };
+
+        private readonly ILogger<WeatherForecastController> _logger;
+
+        // The Web API will only accept tokens 1) for users, and 2) having the access_as_user scope for this API
+        static readonly string[] scopeRequiredByApi = new string[] { "user_impresionation" };
```
Comment (jmprieur) [inline tests/blazorwasm2-b2c-hosted/Server/Controllers/WeatherForecastController.cs:26]: Was it registered as `user_impresionation` in the app registration? or with the typo?

Comment (jennyf19) [reply tests/blazorwasm2-b2c-hosted/Server/Controllers/WeatherForecastController.cs]: oops. idk. i probably mis-typed it. or it's wrong in the app registration. will check.

**Pull request merged**